physlr/physlr.py

```diff
@@ -1239,6 +1239,46 @@ def physlr_overlap(self):
         self.write_tsv(g, sys.stdout)
         print(int(timeit.default_timer() - t0), "Wrote the graph", file=sys.stderr)
 
+    def physlr_filter_overlap(self):
+        "Read a Physlr overlap graph and filter edges."
+
+        edge_weight = []
+
+        at_edges = False
+        overlap_input = open(self.args.FILES[0], "r")
```

Comment:
```suggestion
        with open(self.args.FILES[0], "r") as overlap_input:
```
I prefer that style of opening/working with files

Reply:
the code here is in a separate PR to keep things separate

Reply:
hmm Ok probably it would have been good to not commit the code to this PR then -- that's a bit confusing